### PR TITLE
added observation-realtime-streaming.py script

### PR DESCRIPTION
**Pull request by prashanthShiksha**

### CODE REVIEW

File: observations/observation_realtime_streaming.py [hunk 1]
```diff
@@ -0,0 +1,1108 @@
...
+producer = KafkaProducer(bootstrap_servers=[kafka_url])
+
+#db production
+# client = MongoClient(config.get('MONGO', 'url'))
+# db = client[config.get('MONGO', 'database_name')]
+# solCollec = db[config.get('MONGO', 'solutions_collection')]
+# obsCollec = db[config.get('MONGO', 'observations_collection')]
+# criteriaQuestionsCollec = db[config.get('MONGO', 'criteria_questions_collection')]
+# criteriaCollec = db[config.get('MONGO', 'criteria_collection')]
+# programsCollec = db[config.get('MONGO', 'programs_collection')]
```
Comment (adithyadinesh0412) [inline observations/observation_realtime_streaming.py]: Remove this code @prashanthShiksha

File: observations/observation_realtime_streaming.py [hunk 2]
```diff
@@ -0,0 +1,1108 @@
...
+
+
+# # Define function to check if observation submission Id exists in Druid
+def check_observation_submission_id_existance(key,column_name,table_name):
```
Comment (adithyadinesh0412) [inline observations/observation_realtime_streaming.py]: change the variable name key -> ObservationSubmissionId

File: observations/observation_realtime_streaming.py [hunk 3]
```diff
@@ -0,0 +1,1108 @@
...
+          # Query to check existence of observation submission Id in Druid table
+          query = f"SELECT COUNT(*) FROM \"{table_name}\" WHERE \"{column_name}\" = '{key}'"
+          cur.execute(query)
+          result = cur.fetchone()
+          count = result[0]
+          infoLogger.info(f"Found {count} entires in {table_name}")
+          if count == 0:
+              return True
+          else:
+              return False
```
Comment (adithyadinesh0412) [inline observations/observation_realtime_streaming.py:127]: Check this logic

File: observations/observation_realtime_streaming.py [hunk 4]
```diff
@@ -0,0 +1,1108 @@
...
+      else:
+          # Since the table doesn't exist, return True to allow data insertion initially 
+          return True             
+  except Exception as e:
+      # Log any errors that occur during Druid query execution
+      errorLogger.error(f"Error checking observation_submission_id existence in Druid: {e}")
+   
+def check_datasource_existence(datasource_name):
+  host = config.get('DRUID', 'datasource_url')
```
Comment (adithyadinesh0412) [inline observations/observation_realtime_streaming.py]: Handle this case. 
It might break the execution of the program if config is missed.

File: observations/observation_realtime_streaming.py [hunk 5]
```diff
@@ -0,0 +1,1108 @@
...
+          roleObj = {}
+          roleObj["role_title"] = userSubType
+          roleObj["user_boardName"] = boardName
+          roleObj["user_type"] = user_type
+          #commented it beacause "userLocation" key not present in userProfile
+          # if "userProfile" in obSub and len(obSub["userProfile"]["userLocations"])>0:
+          #  for ent in obSub["userProfile"]["userLocations"]:
+          #     roleObj["user_"+ent["type"]+"Name"] = ent["name"]
+          #     roleObj[ent["type"]+"_externalId"] = ent["id"]
+          #     roleObj[ent["type"]+"_code"] = ent["code"]
```
Comment (adithyadinesh0412) [inline observations/observation_realtime_streaming.py:309]: remove

Comment (adithyadinesh0412) [inline observations/observation_realtime_streaming.py]: @prashanthShiksha Not satisfied with the reason to comment this code out. Please get more clarity and let me know why was this removed

File: start-services.sh [hunk 6]
```diff
@@ -4,7 +4,7 @@
 /opt/sparkjobs/faust_as_service/faust.sh  survey/survey_realtime_streaming survey/ &
 
 #start observations
-#/opt/sparkjobs/faust_as_service/faust.sh  observations/py_observation_streaming observations/ &
```
Comment (Vivek-M-08) [inline start-services.sh:7]: Don't rename the existing one this will be used in sunbird, Please create one more
